Tests/test_imagepath.py

```diff
@@ -39,48 +38,76 @@ def test_path():
     p.transform((1, 0, 1, 0, 1, 1))
     assert list(p) == [(1.0, 2.0), (5.0, 6.0), (9.0, 10.0)]
 
-    # alternative constructors
-    p = ImagePath.Path([0, 1])
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path([0.0, 1.0])
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path([0, 1])
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path([(0, 1)])
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path(p)
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path(p.tolist(0))
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path(p.tolist(1))
-    assert list(p) == [(0.0, 1.0)]
-    p = ImagePath.Path(array.array("f", [0, 1]))
-    assert list(p) == [(0.0, 1.0)]
 
-    arr = array.array("f", [0, 1])
-    p = ImagePath.Path(arr.tobytes())
+@pytest.mark.parametrize(
+    "coords",
+    (
+        (0, 1),
+        [0, 1],
+        (0.0, 1.0),
+        [0.0, 1.0],
+        ((0, 1),),
+        [(0, 1)],
+        ((0.0, 1.0),),
+        [(0.0, 1.0)],
+        array.array("f", [0, 1]),
+        array.array("f", [0.0, 1.0]),
+        ImagePath.Path((0, 1)),
```

Comment:
```suggestion
        ImagePath.Path(array.array("f", [0, 1]).tobytes()),
```
A suggestion, to keep the original test case of
```python
arr = array.array("f", [0, 1])
p = ImagePath.Path(arr.tobytes())
```
intact.

Reply:
Line 54 should change, not 55. The new line 55 is for the original line 51.

Reply:
Right, thanks